.jenkins/pytorch/test.sh

```diff
@@ -308,12 +335,15 @@ elif [[ "${BUILD_ENVIRONMENT}" == *-test2 || "${JOB_BASE_NAME}" == *-test2 ]]; t
   test_libtorch
   test_custom_script_ops
   test_torch_function_benchmark
+elif [[ "${BUILD_ENVIRONMENT}" == *-test3 || "${JOB_BASE_NAME}" == *-test3 ]]; then
```

Comment:
`JOB_BASE_NAME`? If this is to deal with Jenkins differences, don't you have to apply this everywhere else?